Library/Homebrew/cmd/list.rb

```diff
@@ -69,45 +75,52 @@ def list_args
   def list
     args = list_args.parse
 
-    return list_casks(args: args) if args.cask?
-
     if args.unbrewed?
       raise UsageError, "`--unbrewed` does not take a formula/cask argument." unless args.no_named?
 
-      return list_unbrewed
+      odeprecated "`brew list --unbrewed`", "`brew --prefix --unbrewed`"
```

Comment:
```suggestion
      odisabled "`brew list --unbrewed`", "`brew --prefix --unbrewed`"
```

unless this logic is passed through to `brew --prefix --unbrewed`.